Makefile

```diff
@@ -1,12 +1,22 @@
 all: test
 
-uninstall:
-	pip freeze | grep -v "^-e" | xargs pip uninstall -y
-
-install:
+install.tools:
 	pip install --upgrade pip twine wheel
+
+## Install openfisca-core for deployment and publishing.
+install.build: setup.py clean install.tools
+	@## This allows us to be sure tests are run against the packaged version
+	@## of openfisca-core, the same we put in the hands of users and reusers.
+	pip install --upgrade pip wheel
```

Comment:
This seems to repeat `install.tools` while that target is a dependency of this one. What do we gain from splitting `install` in two if in the end `install.tools` is a dependency of both `install.build` and `install` and repeated in `install.build`?

Reply:
Yes, I agree, #1015 adresses this already.

Reply:
Fixed.